frontend/src/index.js

```diff
@@ -1,4 +1,3 @@
-import "babel-polyfill";
```

Comment:
That's for IE compatibility.  Is that not needed anymore?

Reply:
It looks like IE compatability is broken at the moment. I'll see if I can get it working again.

Reply:
Before you dive too deep into that, i'd recommend asking the business if they actually care.
And if it takes too long, focus on fixing everything else with this PR and ship it as long as it works in every other browser 👌 
Then look at fixing IE

Reply:
If it was a somewhat graceful break I would agree, but this is just a whitescreen load. If I can't fix it I'll try and shim a compatibility error message in front. I asked the PO about it and he said to hold off bringing it in until we addressed it.